OpenLIFUHome/OpenLIFUHome.py

```diff
@@ -212,6 +228,9 @@ def getParameterNode(self):
     def clear_session(self) -> None:
         self.current_session = None
     
+    def toggle_guided_mode_status(self):
```

Comment:
perhaps `toggle_guided_mode_state` is better? since it's not a status display but rather internal application state

perhaps there could be an optional `bool` input for whether we want to set the state to on or off? in that case the name could be `set_guided_mode_state` and the docstring can explain that if not provided it acts as a toggle

the reason I suggest this is that the custom app wants to be 100% certain is turning _on_ guided mode, and calling this toggle feels a bit iffy